setup.py

```diff
@@ -22,16 +22,17 @@
 MAJOR = 6
 MINOR = 0
 MICRO = 0
-IS_RELEASED = False
+PRERELEASE = "rc0"
```

Comment:
For non-prerelease is this going to be `""`?  Do these changes to setup.py need to be pushed across to master (or should the changes to setup.py be applied to master and then backported)?

Reply:
> For non-prerelease is this going to be `""`?

Yes.

> Do these changes to setup.py need to be pushed across to master

Yes. There's an entry in the release checklist with a reminder to do this.

Reply:
I cherry-picked the middle commit across to master in #864.